homepage/tests/test_views.py

```diff
@@ -43,7 +43,7 @@
 
 def test_homepage(db, tp, django_assert_num_queries):
     url = tp.reverse("home")
-    with django_assert_num_queries(10):
+    with django_assert_num_queries(15):
```

Comment:
@jefftriplett is the increase in expected number of queries because of the introduction of the use of waffle?

Reply:
I believe so. There's an option to use redis (I haven't read it in a while) but that's kind of a moot point when testing it. We can test with the flags off and on to narrow it down, but I was mostly looking for a way to ship your feature before dependabot starts burying us in updates :)